torch/onnx/symbolic_opset10.py

```diff
@@ -377,8 +377,11 @@ class Quantized:
 
     @staticmethod
     def linear(g, q_input, q_weight, bias, op_scale, op_zero_point):
-        input, _, _ = sym_help.dequantize_helper(g, q_input)
-        weight, _, _ = sym_help.dequantize_helper(g, q_weight)
+        input, input_scale, _ = sym_help.dequantize_helper(g, q_input)
+        weight, weight_scale, _ = sym_help.dequantize_helper(g, q_weight)
+        # Preserve bias quantization step in ONNX graph
```

Comment:
I think if you have a good comment in the doc string for `requantize_bias_helper`, then we don't need a comment above every call to that function.